pom.xml

```diff
@@ -111,7 +111,7 @@
         </dependency>
         <dependency>
             <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk-ec2</artifactId>
+            <artifactId>aws-java-sdk-ecs</artifactId>
             <version>1.10.51</version>
```

Comment:
@xupyprmv sorry for barging in here unannounced already, but can you please up all the AWS versions to `1.10.66` while we're at it here :)
No need to keep this version locked for any reason, the previous use of it was simply unused in practice so we should start from the newest here :) thanks!